client/src/components/home/Home.test.js

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
```

Comment:
is there a difference between `import * as React from 'react'` vs `import React from 'react'` If not, should stick with one